contrib-viz/03.contrib-stats.ipynb

```diff
@@ -0,0 +1,481 @@
+     ]
+    }
+   ],
+   "source": [
```

Comment:
Do we need this block?  We already have different ways to track the number of authors that might conflict with this count.

Reply:
Removed!